src/server/test/config.ini

```diff
@@ -493,15 +493,15 @@ profiler::cancelled = false
 ;profiler::timeout.qps = false
 
 [meta_server]
-server_list = 0.0.0.0:34701
+server_list = 127.0.0.1:34701
```

Comment:
Why change this?
If the default behavior has changed to treat the server_list as hostnames, and it doesn't work if they are IP addresses, how about introducing a new config, e.g. `hostname_server_list`, to make it compatiable when user upgrade Pegasus from old versions, so they don't needed to change configs when upgrade.

Other places are the same.